app/containers/Login/saga.js

```diff
@@ -137,9 +138,15 @@ export function* loginWithScatterWorker() {
     }
 
     if (!eosService.selectedAccount) {
-      throw new WebIntegrationError(
-        translations[messages[USER_IS_NOT_SELECTED].id],
-      );
+      if (eosService.isScatterExtension) {
```

Comment:
I would be nice to refactor this code a bit to avoid duplicating logic 3 times.

Maybe add a method `checkAccountSelected` to eos service?

Reply:
OK, I'll check.